package/yast2-network.changes

```diff
@@ -1,4 +1,11 @@
 -------------------------------------------------------------------
+Tue Jun 21 05:37:31 UTC 2016 - igonzalezsosa@suse.com
+
+- bsc#984146
+  - set AutoYaST keep_install_network parameter to 'yes' by default
```

Comment:
The value is actually *true*. Let's use that since other config files allow 'yes' and we should not add to the confusion.